packages/react-app/src/pages/manage.js

```diff
@@ -1,25 +1,54 @@
 import React from 'react'
 import { useState, useEffect } from 'react';
 import { Container, Row, Col } from 'react-bootstrap/';
-import { Grey, Blue, InputBox, ButtonBox, PrimaryButton, CircleQ, WorkerRunwayDisplay, DataRow, SecondaryButton } from '@project/react-app/src/components'
+import { Grey, Blue, InputBox, ButtonBox, PrimaryButton, CircleQ, WorkerRunwayDisplay, DataRow, SecondaryButton, EthBalance, NuBalance} from '@project/react-app/src/components'
 import useWeb3Modal from '../hooks/useWeb3Modal'
 
 export function Manage() {
 
-    const [availableNU, setavailableNU] = useState(0);
-    const [availableETH, setavailableETH] = useState(0);
+    const [availableNU, setAvailableNU] = useState(0);
+    const [availableETH, setAvailableETH] = useState(0);
+
+    const [availableNuRewards, setAvailableNuRewards] = useState(0);
+    const [availableEthRewards, setAvailableEthRewards] = useState(0);
 
     const [workerAddress, setWorkerAddress] = useState(null);
     const [stakerAddress, setStakerAddress] = useState(null);
 
-    const [stakeList, setStakeList] = useState([{start: '2021-04-01',  end: '2021-05-19', lockedNU: 48000}, {start: '2021-04-01', end:  '2021-07-19', lockedNU: 96000}]);
+    const [stakerData, setStakerData] = useState({substakes:[]});
+
+    const [provider, loadWeb3Modal, logoutOfWeb3Modal, account, web3, contracts] = useWeb3Modal()
 
-    const [provider, loadWeb3Modal, logoutOfWeb3Modal, account] = useWeb3Modal()
+    useEffect(() => async () => {
+        if (contracts && account){
+            const stakerInfo = await contracts.STAKINGESCROW.methods.stakerInfo(account).call()
+            const stakerFlags = await contracts.STAKINGESCROW.methods.getFlags(account).call()
+            const getSubStakesLength = await contracts.STAKINGESCROW.methods.getSubStakesLength(account).call();
```

Comment:
@KPrasch have a look at how this works.
contracts is a value that gets set by the web3modal/button thing.
It will always be set [here](https://github.com/nucypher/nuit/pull/14/files#diff-196bdddfadeecacb9b3b1c903c8ff441b35ac5e6a0597b2b4a7a097113dfc002R50) for the correct chainID: